Added: alt attribute to few images (#19669)
mdn/content: files/en-us/learn/javascript/client-side_web_apis/client-side_storage/index.md

The app looks something like this:

> ![A screenshot of a website that has 4 sections. The first section is the header that reads 'IndexedDB notes demo' which is the name of the exercise. The second section lists all the notes that are created. It has two notes and each note has a delete button. The third section has 2 input fields with labels 'Note title' and 'Note text' respectively and a button that is labelled 'Create new note'. The last section is a footer that reads 'Copyright nobody. Use the code as you like'.](idb-demo.png)

Reviewer: ```suggestion
![IndexDB notes demo screenshot with 4 sections. The first section is the header. The second section lists all the notes that have been created. It has two notes, each with a delete button. A third section is a form with 2 input fields for 'Note title' and 'Note text' and a button labeled 'Create new note'. The bottom section footer reads 'Copyright nobody. Use the code as you like'.](idb-demo.png)
```

The above example already shows how to create an app that will store large assets in an IndexedDB database, avoiding the need to download them more than once. This is already a great improvement to the user experience, but there is still one thing missing — the main HTML, CSS, and JavaScript files still need to be downloaded each time the site is accessed, meaning that it won't work when there is no network connection.

> ![There is an illustration of a fictious cartoon character to the left-hand side. The character is holding a two pin plug in it's right hand and a two pin plug holder in it's left hand. The right-hand side displays a message saying that the user is in offline mode and there is a button to the right-hand bottom corner that is labelled 'Try again'.](ff-offline.png)

Reviewer: ```suggestion
![Firefox offline screen with an illustration of a cartoon character to the left-hand side holding a two-pin plug in its right hand and a two-pin socket in its left hand. On the right-hand side there is an Offline Mode message and  a button labeled 'Try again'.](ff-offline.png)
```
